cluster/examples/kubernetes/ceph/operator-openshift.yaml

```
@@ -154,6 +154,10 @@ data:
 
   # (Optional) Allow starting unsupported ceph-csi image
   ROOK_CSI_ALLOW_UNSUPPORTED_VERSION: "false"
+
+  # (Optional) control the host mount of /etc/selinux for csi plugin pods.
+  # CSI_PLUGIN_ENABLE_SELINUX_HOST_MOUNT: "true"
```

This is the same default, but perhaps easier to see the default than when it is commented out. 
```suggestion
  CSI_PLUGIN_ENABLE_SELINUX_HOST_MOUNT: "false"
```